auth_otp/services/kavenegar_service.py

```
@@ -0,0 +1,232 @@
+            )
+            
+            return {
+                'success': True,
+                'message_id': str(response['messageid']),
+                'status': response['status'],
+                'status_text': response['statustext']
+            }
+            
+        except APIException as e:
+            # خطای API کاوه‌نگار
+            logger.error(f"Kavenegar API error: {e}")
```

Ruff (0.12.2)

Use `logging.exception` instead of `logging.error`

Replace with `exception`

(TRY400)